src/ParkMap.js

```diff
@@ -50,11 +75,26 @@ class ParkMap extends Component {
             this.props.parks.map(park => {
               let lat = (park.latitude.includes('N')) ? park.latitude.replace(/.N$/, '') : park.latitude.replace(/.S$/, '').replace(/^/, '-');
               let lon = (park.longitude.includes('E')) ? park.longitude.replace(/.E$/, '') : park.longitude.replace(/.W$/, '').replace(/^/, '-');
-              return(
-                <Marker position={[lat, lon]} onClick={this.getPark} key={park.urlCode} id={park.urlCode}>
-                  <Tooltip>{park.parkName}</Tooltip>
-                </Marker>
-              )
+              if (this.props.visitedParks.includes(park.urlCode)) {
```

Comment:
Awesome job getting the different icon colors to display when it's a visited or bucket list park! Really like how the map looks now when a user has visited and bucket list parks